src/mongo_logger.ts

```diff
@@ -576,7 +576,7 @@ function defaultLogTransform(
       log.message = 'Server heartbeat succeeded';
       log.durationMS = logObject.duration;
       log.serverConnectionId = logObject.serverConnectionId;
-      log.reply = stringifyWithMaxLen(logObject.reply, maxDocumentLength);
+      log.reply = EJSON.stringify(logObject.reply, { relaxed: true });
```

Comment:
What is the reason for this change?

Reply:
[According to the spec the reply field has to be relaxed](https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring-logging-and-monitoring.rst#server-heartbeat-succeeded-log-message), which according to our [Node Logging Design Document](https://www.google.com/url?sa=t&rct=j&esrc=s&source=appssearch&uact=8&cd=0&cad=rja&q&sig2=HXozYfPozt3MApq7JZxyGQ&ved=0ahUKEwjDorKzr6OCAxUQwDcKHXsPBkE4ABABKAAwAA&url=https://drive.google.com/a/mongodb.com/open?id%3D1L3onvBqFaXuiN21sXMaFBRNS5kPnNYYckomt0s0ThmY&usg=AOvVaw0M0bd9q-Eh4qRu3zch2fhT), it will not be truncated.

Reply:
nvm, just extended maxDocLength in spec runner, ^^ above is not true.